.github/workflows/issues_and_pull_requests.yml

```
@@ -11,7 +11,7 @@ on:
 env:
   CLOSE_MESSAGE: >-
     The fact this was automatically closed doesn't mean that the idea got rejected - it simply didn't get any priority for way too long to keep it open.
-    If you're still interested in this, please let as know, we can consider re-opening it.
+    If you are interested in implementing this, please let as know.
```

```suggestion
    If you are interested in implementing this, just do it.
```

Although it says: let us know IF one is interested in IMPLEMENTING this, I can easily imagine people posting with a message like "hi, I'm interested with this" - meaning with someone else making it.

Can we phrase it similar to what I've suggested, but more politely?

---

it does not say "if one is interested". It says "if you are interested"

anyway,
```suggestion
    If you are interested in implementing this yourself, please share the implementation draft to align on the approach, before you prepare the full implementation.
```